package.json

```diff
@@ -80,6 +83,7 @@
     "@types/serve-handler": "6.1.1",
     "@types/uuid": "8.3.4",
     "@typescript-eslint/eslint-plugin": "5.38.0",
+    "@typescript-eslint/parser": "^5.38.0",
```

Comment:
I needed to add this otherwise my eslint was failing, because it couldn't find `@typescript-eslint/parser`